eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -574,6 +574,10 @@ the main ServiceBusClientBuilder. -->
   <suppress checks="com.azure.tools.checkstyle.checks.OnlyFinalFieldsForImmutableClass"
             files="com.azure.ai.formrecognizer.models.(TextAppearance|RecognizedForm|FormTable|FormSelectionMark|FormPage|FormLine).java"/>
 
+  <!-- Suppress for the OnlyFinalFieldsForImmutableClass on Text Analytics -->
+  <suppress checks="com.azure.tools.checkstyle.checks.OnlyFinalFieldsForImmutableClass"
+            files="com.azure.ai.textanalytics.(models|util).*\.java"/>
```

Comment:
I think the general suggestion is to specify the classes requiring this suppression rather than suppressing all from the folder.
Would that be reasonable, though?

Reply:
Too many classes to suppression. I think wildcard is useful here?